### PR TITLE
Fix-pe-dry-run

### LINKED ISSUE #126: [PE-D][Tester D] Inconsistent error messages for deletion
There seems to be an inconsistency in the error message given for two large numbers. Could possibly standardise the messages between the two large numbers.

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: yaojiax/ped#2

### LINKED ISSUE #128: [PE-D][Tester D] Wrong error message
Provided a positive number but the error message suggests that I did not give a positive number. Could possibly check for cases of integer overflow and give it a more helpful error message. 

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: yaojiax/ped#3

### LINKED ISSUE #129: [PE-D][Tester E] Titles with all capital letters fails.
Entering a title with all capital letters result in error message but should be accepted as titles are specified to be alphanumeric characters.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: zenithyap/ped#1

### LINKED ISSUE #139: [PE-D][Tester B] Autocomplete: Autocomplete feature works for most commands but does not work for sort command
I understand that the autocomplete feature is to use tab (After entering v, a “tab to cycle through suggestions” pops up)

After clicking tab

But this does not work for the Sort command

If I click tab, nothing happens

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: potty10/ped#3

### PR DESCRIPTION
**Pull request by yyj-02**
## Which ticket does this address

- closes #139
- closes #129
- closes #128
- closes #126

## What problems does this solve

- Many many bugs

## Are there any breaking change

- alls good

## Checklist

- [ ] All tests pass locally and on CI

### CODE REVIEW

File: docs/tutorials/AddRemark.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Following the convention in other commands, we add relevant messages as constant
             + ": Edits the remark of the person identified "
             + "by the index number used in the last person listing. "
             + "Existing remark will be overwritten by the input.\n"
-            + "Parameters: INDEX (must be a positive integer) "
+            + "Parameters: INDEX (must be a positive integer within the range of the number of listings shown) "
```
Comment (clevon-w) [inline docs/tutorials/AddRemark.md:71]: Are these changes reflected in the UG? Also how come codecov so low

Comment (clevon-w) [reply docs/tutorials/AddRemark.md:71]: @panpannnnn @nt-nic for the UG part

**Review by clevon-w — COMMENTED** [review]
lgtm other than UG changes and testcases changes, not sure if yall know about them if yall know and plan on fixing in another pr i will go ahead and merge

**Review by homuntan02 — APPROVED** [review]
Lgtm

**Pull request merged**